sdk/eventgrid/azure-eventgrid/tests/perfstress_tests/README.md

```diff
@@ -0,0 +1,49 @@
+# EventGrid Performance Tests
+
+In order to run the performance tests, the `azure-devtools` package must be installed. This is done as part of the `dev_requirements`.
+Start by creating a new virtual environment for your perf tests. This will need to be a Python 3 environment, preferably >=3.7.
+
+### Setup for test resources
+
+These tests will run against a pre-configured Eventgrid topic. The following environment variable will need to be set for the tests to access the live resources:
+```
+EG_ACCESS_KEY=<access key of your eventgrid account>
+EG_TOPIC_HOSTNAME=<hostname of the eventgrid topic>
+```
+
+### Setup for perf test runs
+
+```cmd
+(env) ~/azure-eventgrid> pip install -r dev_requirements.txt
+(env) ~/azure-eventgrid> pip install -e .
+```
+
+## Test commands
+
+```cmd
+(env) ~/azure-eventgrid> cd tests
+(env) ~/azure-eventgrid/tests> perfstress
+```
+
+### Common perf command line options
+These options are available for all perf tests:
+- `--duration=10` Number of seconds to run as many operations (the "run" function) as possible. Default is 10.
+- `--iterations=1` Number of test iterations to run. Default is 1.
+- `--parallel=1` Number of tests to run in parallel. Default is 1.
+- `--no-client-share` Whether each parallel test instance should share a single client, or use their own. Default is False (sharing).
```

Comment:
You can remove this option unless you want to implement sharing a single client between test instances.
Probably not needed for now.